modin/pandas/test/test_io.py

```diff
@@ -437,8 +505,11 @@ def test_read_csv_nans_handling(
         verbose,
         skip_blank_lines,
     ):
```

Comment:
Why did you change `test_read_csv_nans_handling` test?

Reply:
There is no actual reason for this, changes were rollbacked.